poglink/cogs/rates.py

```diff
@@ -105,10 +107,14 @@ async def send_embed(self, description, url):
             return
 
         server_name = server_meta.get("short_name")
+        
+        # generate dynamic timestamp (https://hammertime.djdavid98.art/)
+        ts = int(datetime.datetime.now().timestamp()//60 * 60)
```

Comment:
Added this to generate timestamp string as per discord's [dynamic timestamp format](https://hammertime.djdavid98.art/)

Reply:
Any reason why this isn't just `time.time()` ?

Reply:
Does this need to be rounded to the nearest minute?

Reply:
Switched to `time.time()`.

It doesn't need to be rounded to minutes, discord accepts [unix timestamp in seconds](https://discord.com/developers/docs/reference#:~:text=%3Ct%3A-,1618953630,-%3E) and handles rounding